test/screenshot/stop.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-kill $(ps aux | grep 'webpack' | awk '{print $2}') || echo 'already stopped'
+kill $(ps aux | grep 'webpack' | grep 'screenshot' | awk '{print $2}') || echo 'already stopped'
```

Comment:
Is this so that it also kills the npm screenshot tests running?

Reply:
it should not be there, I'll remove it tomorrow and will make another PR correctly. Currently it kills every webpack project running, including my project